### PR TITLE
[PATCH v5] Tmfinal docupdate2

### PR DESCRIPTION
**Pull request by Bill-Fischofer-Linaro**
Update User Guide to correct typos and bring up to date with Tiger Moth API changes.

### CODE REVIEW

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v2: 
* Fix issues merge conflicts with master

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v4:
* Rebased to latest master
* Incorporate additional typographic corrections contributed by @joseppc

File: doc/users-guide/users-guide-packet.adoc [hunk 1]
```diff
@@ -499,3 +499,38 @@ Note also that a packet may not reference itself, nor may circular reference
...
+processing. However, the application can trigger parsing explicitly via the
+API:
+[source,c]
+-----
+int odp_packet_parse(odp_packet_t pkt, uint32_t offset,
+		     const odp_packet_parse_param_t *param);
+-----
+This is typically done following packet decapsulation or other preprocessing
+that would prevent RX parsing from "seeing" the relevant portion of the
+packet. The `odp_packet_parse_param_t` struct that is passed to controls the
```
Comment (muvarov) [inline doc/users-guide/users-guide-packet.adoc]: to control

**Comment by joseppc** [comment]
Reviewed-by: Josep Puigdemont <<josep.puigdemont@linaro.org>>

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**